.github/workflows/ci.yml

```diff
@@ -18,9 +18,13 @@ on:
   push:
     branches:
       - 'main'
+    paths-ignore:
+      - 'terraform/**'
```

Comment:
I am hoping that the workflow is skipped here so the required status checks still allow PRs to be merged. I am not sure that will be the case, but wanted to call it out just in  case.